<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaEmprestimo</class>
 <widget class="QDialog" name="telaEmprestimo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BIBLIOTECA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivos.qrc">
    <normaloff>:/images/images/logo.ico</normaloff>:/images/images/logo.ico</iconset>
  </property>
  <widget class="QPushButton" name="regsEmprestimo_bt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font-size: 20px;
border-radius: 5px;
background-color:#9d66d6;
color: #fefefe;
font-family:Franklin Gothic Heavy;}

:hover{
background-color:#ad7fdd;}
</string>
   </property>
   <property name="text">
    <string>REGISTRAR</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font-size: 13px;
border-radius: 5px;
background-color:#9d66d6;
color: #fefefe;
font-family:Franklin Gothic Heavy;}

:hover{
background-color:#ad7fdd;}</string>
   </property>
   <property name="text">
    <string>&lt; Voltar</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>451</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#9d66d6;
color:#fefefe;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="login_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:Franklin Gothic Heavy;
font-size:30px;
color:#fefefe;
</string>
    </property>
    <property name="text">
     <string>EMPRESTIMO</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>371</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
       </property>
       <property name="text">
        <string>CLIENTE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="clientes_cbox">
       <property name="minimumSize">
        <size>
         <width>188</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
       </property>
       <property name="text">
        <string>LIVRO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="livros_cbox">
       <property name="minimumSize">
        <size>
         <width>188</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
       </property>
       <property name="text">
        <string>DATA DO EMPRESTIMO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="emprestimo_date">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="arquivos.qrc">:/images/images/444.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>regsEmprestimo_bt</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources>
  <include location="arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
